Source/WebKit/UIProcess/WebScreenOrientationManagerProxy.cpp

```diff
@@ -150,13 +150,13 @@ void WebScreenOrientationManagerProxy::unlock()
 
     bool didUnlockOrientation = false;
 #if ENABLE(FULLSCREEN_API)
-    if (m_page.fullScreenManager() && m_page.fullScreenManager()->isFullScreen()) {
-        m_page.fullScreenManager()->unlockFullscreenOrientation();
+    if (m_page->fullScreenManager() && m_page->fullScreenManager()->isFullScreen()) {
+        m_page->fullScreenManager()->unlockFullscreenOrientation();
         didUnlockOrientation = true;
     }
 #endif
     if (!didUnlockOrientation)
-        m_page.uiClient().unlockScreenOrientation(m_page);
+        m_page->uiClient().unlockScreenOrientation(m_page.get());
```

Comment:
We likely need to pass a `protectedPage()` to `unlockScreenOrientation()`

Reply:
sure, fixed.